fastparquet/api.py

```diff
@@ -500,6 +501,8 @@ def get_type(name):
     dtypes.extend(['category'] * len(cs))
     df, views = dataframe.empty(dtypes, size, cols=cols, index_name=index,
                                 index_type=index_type, cats=cats)
+    if index and re.match(r'__index_name_\d+__', index):
```

Comment:
This should be `__index_level_\d+__`. Sorry I wrote the wrong thing in my comment.